package.json

```diff
@@ -1,85 +1,79 @@
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/boardgameio/storage-cache.git"
-  },
-  "bugs": {
-    "url": "https://github.com/boardgameio/storage-cache/issues"
-  },
-  "keywords": [
-    "boardgame.io",
-    "storage",
-    "cache"
-  ],
-  "funding": [
-    "https://github.com/boardgameio/boardgame.io?sponsor=1",
-    {
-      "type": "opencollective",
-      "url": "https://opencollective.com/boardgameio"
+    "name": "@boardgame.io/storage-cache",
+    "version": "0.3.3",
+    "description": "🔌 A caching layer for boardgame.io database connectors",
+    "license": "MIT",
+    "author": "Chris Swithinbank <swithinbank@gmail.com> (http://chrisswithinbank.net/)",
+    "homepage": "https://github.com/boardgameio/storage-cache#readme",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/boardgameio/storage-cache.git"
+    },
+    "bugs": {
+        "url": "https://github.com/boardgameio/storage-cache/issues"
+    },
+    "keywords": [
+        "boardgame.io",
+        "storage",
+        "cache"
+    ],
+    "funding": [
+        "https://github.com/boardgameio/boardgame.io?sponsor=1",
+        {
+            "type": "opencollective",
+            "url": "https://opencollective.com/boardgameio"
+        }
+    ],
+    "main": "dist/bgio-storage-cache.js",
+    "types": "dist/bgio-storage-cache.d.ts",
+    "files": [
+        "dist/*"
+    ],
+    "scripts": {
+        "pretest": "eslint \"{src,test}/**/*.{ts,js}\"",
+        "test": "jest --coverage",
+        "coverage": "cat ./coverage/lcov.info | coveralls",
+        "prettier:check": "prettier --list-different \"{src,test}/**/*.ts\"",
+        "prettier:write": "prettier --write \"{src,test}/**/*.ts\"",
+        "clean": "rimraf dist .tscache",
+        "prebuild": "npm run prettier:check && npm run clean",
+        "build": "tsc -b src",
+        "prerelease": "npm run test && npm run build",
+        "release": "standard-version -s",
+        "prepublishOnly": "npm run build"
+    },
+    "config": {
+        "commitizen": {
+            "path": "./node_modules/cz-conventional-changelog"
+        }
+    },
+    "devDependencies": {
+        "@types/jest": "^27.0.1",
+        "@typescript-eslint/eslint-plugin": "^2.25.0",
+        "@typescript-eslint/parser": "^2.25.0",
+        "boardgame.io": "^0.49.0",
+        "commitizen": "^4.2.1",
+        "coveralls": "^3.0.11",
+        "cz-conventional-changelog": "^3.1.0",
+        "eslint": "^6.8.0",
+        "eslint-config-prettier": "^6.10.1",
+        "eslint-plugin-jest": "^25.0.5",
+        "eslint-plugin-prettier": "^3.1.2",
+        "jest": "^27.1.0",
+        "node-persist": "^3.0.5",
+        "prettier": "^2.0.2",
+        "rimraf": "^3.0.2",
+        "standard-version": "^9.0.0",
+        "tempy": "^1.0.0",
+        "ts-jest": "^27.0.5",
+        "typescript": "^4.2.4"
+    },
+    "dependencies": {
+        "@types/lru-cache": "^5.1.0",
+        "husky": "^7.0.4",
```

Comment:
@Stefan-Hanke Thanks for taking this on!

I think husky should be installed as a dev dependency rather than here — package consumers won't need it.

I'm on my phone so it's hard to see exactly, but it looks like maybe `package.json` was reformatted using tabs instead of spaces? If you could revert that, it would be easier to see the significant changes.

Reply:
Ahh all those VSCode magic on save. Definitely need to check how to configure that. Thanks for the hints!
Force-pushed since the old commit is not relevant anymore, diff looks straight now.